Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/NonStandardOptionsTests.cs

```diff
@@ -223,7 +223,7 @@ public void ValidateCachingUnsafeUnexpectedFileAccessesAreErrorsOutput()
             AssertWarningEventLogged(EventId.ProcessNotStoredToCacheDueToFileMonitoringViolations, count: 2);
         }
 
-        [FactIfSupported(requiresJournalScan: true)]
+        [FactIfSupported(requiresWindowsBasedOperatingSystem: true, requiresJournalScan: true)]

```

Comment:
Is it flakey because it runs on Mac? But how can it run on Mac although it's marked to require journal scan?

I saw this test failing on Windows OS. So is this fix a real fix or an attempt?